### PR TITLE
Fix misspelling issues using locale US

### PR DESCRIPTION
**Pull request by mjtrangoni**
Hi @xlson @torkelo,

related to #14205, I fixed some or most of the issues, using `locale US`.
The final command line with many excludes was,

```sh
$ find . -type f | xargs misspell -locale US | grep -vi -e vendor -e node_modules -e devenv -e unknwon -e destory -e yarn -e adn -e Spindel
```

I have to TAL how can we use `misspell` for checking this directly at CircleCI and not using grep.

### CODE REVIEW

File: public/app/plugins/datasource/prometheus/mode-prometheus.js [hunk 1]
```diff
@@ -433,12 +433,12 @@ var PrometheusCompletions = function() {};
 exports.PrometheusCompletions = PrometheusCompletions;
 });
 
-ace.define("ace/mode/behaviour/prometheus",["require","exports","module","ace/lib/oop","ace/mode/behaviour","ace/mode/behaviour/cstyle","ace/token_iterator"], function(require, exports, module) {
+ace.define("ace/mode/behavior/prometheus",["require","exports","module","ace/lib/oop","ace/mode/behavior","ace/mode/behavior/cstyle","ace/token_iterator"], function(require, exports, module) {
```
Comment (bergquist) [inline public/app/plugins/datasource/prometheus/mode-prometheus.js]: I think updating those imports will break.

**Comment by bergquist** [comment]
I think PR's like this should be smaller. Would be great to start with just .md files, then `pkg/**`, then `public/**` etc

File: public/app/plugins/datasource/prometheus/mode-prometheus.js (hunk 1 above)
Comment (mjtrangoni) [reply public/app/plugins/datasource/prometheus/mode-prometheus.js]: Ok, I will be taking this into account.

**Comment by mjtrangoni** [comment]
@bergquist Ok, no problem. I will be dropping it today.

**Comment by mjtrangoni** [comment]
@bergquist done. PTAL

**Comment by bergquist** [comment]
Thank you for contributing!

**Pull request merged**